source/content/integrated-composer.md

```diff
@@ -0,0 +1,162 @@
+---
+title: Integrated Composer
+description: Learn how to deploy a site with Integrated Composer
+tags: [composer, workflow]
+categories: [get-started]
+contributors: [ari, edwardangert]
+reviewed: "2020-11-18"
+---
+
+Integrated Composer lets you deploy your site on Pantheon with one-click updates for both upstream commits and Composer dependencies, while still receiving upstream updates.
+
+Create a new site with Integrated Composer as part of Pantheon's Limited Availability release. New sites created through Pantheon's Limited Availability program are production-ready.
+
+## Create a New Site With Integrated Composer
+
+### Drupal 9 with Integrated Composer
+
+Please note the Limited Availability program does not include a path to upgrade from previous Drupal versions to Drupal 9. Upgrade instructions for existing Drupal 8 Composer-enabled sites will be available when Integrated Composer moves into General Availability.
+
+<Partial file="drupal-9-upstream-install.md" />
+
+### WordPress with Integrated Composer
+
+1. [Fork the Pantheon-maintained repository](/create-custom-upstream#create-and-host-the-repository-remotely) from [https://github.com/pantheon-upstreams/wordpress-project](https://github.com/pantheon-upstreams/wordpress-project).
```

Comment:
Any reason not to use a `https://dashboard.pantheon.io/sites/create?upstream_id=` -style URL for WordPress with IC, from a product point of view? Making your own fork and custom upstream isn't a requirement technically.